SoundSwitch/Framework/Profile/ProfileSetting.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using SoundSwitch.Common.Framework.Audio.Device;
+using SoundSwitch.Common.WinApi.Keyboard;
+
+namespace SoundSwitch.Framework.Profile
+{
+#nullable enable
+    public class ProfileSetting : IEquatable<ProfileSetting>
+    {
+        public string ProfileName { get; set; } = "";
+        public string? ApplicationPath { get; set; }
+        public HotKeys? HotKeys { get; set; }
+        public DeviceInfo? Playback { get; set; }
```

Comment:
Because I need more than just the ID, I need also the Type of device, I prefer the keep the full DeviceInfo.

There can't be any duplicate Id, or duplicate profile in the configuration since I'm using a hashset with ProfileSetting.

It's also make the UI binding easier.